### DISCUSSION TITLE
Video that explains how to get each one

### DISCUSSION THREAD

**Discussion by vagnermengali** [Schweinepriester/github-profile-achievements#549] [Q&A]
Can anyone tell me if there's a video with the step by step on how to get it?

**Comment by PSGitHubUser1** [comment]
The badges and how to obtain them:

 1. Pair Extraordinaire - Be a Co-author in a merged pull request [MPR].
 2. Quickdraw - Close an issue _(in the repos which are created by you)_ or a pull request under 5 minutes of opening.
 3. Starstruck - Get at least 16 stars in a repo which is created by you.
 4. Galaxy Brain - Answer other's questions and get 2 accepted answers.
 5. Pull Shark - Merge at least 2 pull requests from other branch to your main branch.
 6. YOLO - Merge a pull request without reviewing the code. (Do this only with your trust one)
 7. Arctic Code Vault Contributor - Cannot be obtained anymore.
 8. Public Sponsor - Sponsor a open source work [repo] via [GitHub Sponsor](https://github.com/sponsors)
 9. Mars 2020 Contributor - Not obtainable anymore.
 10. Heart On Your Sleeve - No info available in public. _(I mean how to obtain)_
 11. Open Sourcerer - No info available in public. _(I mean how to obtain)_ 
         [Recommendation to dev: Make this obtainable (Open Sourcerer) when anyone makes more than 25 repositories]

**Comment by cybersaksham** [comment]
> Can anyone tell me if there's a video with the step by step on how to get it?

Hey, you can refer below video to understand how to get these achievement badges.
https://www.youtube.com/watch?v=wBW-vjM7ccA

**Comment by Tynab** [comment]
To earn the Open Sourcerer achievement on GitHub, you need to make a qualifying contribution to an open source project on GitHub. Here are the steps you can follow:
1. Sign up for a GitHub account if you haven't already.
2. Find an open source project on GitHub that you would like to contribute to. You can search for open source projects using GitHub's Explore feature, or you can look for projects in specific programming languages or topics that interest you.
3. Fork the repository of the open source project that you want to contribute to. This creates a copy of the project in your own GitHub account that you can make changes to.
4. Make a qualifying contribution to the project. This can be anything from fixing a bug or adding a new feature, to improving documentation or submitting a new translation. Your contribution must be accepted and merged into the main repository by the project maintainers to count towards the Open Sourcerer achievement.
5. Once your contribution has been merged, you should receive a notification from GitHub that your contribution has been accepted. After 30 days of making your first contribution, you will earn the Open Sourcerer badge on your GitHub profile.
Note that GitHub has specific criteria for what constitutes a qualifying contribution, so be sure to review the contribution guidelines for the project you want to contribute to before getting started. Good luck!

**Reply by Schweinepriester** [reply]
https://github.com/Schweinepriester/github-profile-achievements/discussions/574#discussioncomment-5011674

**Comment by Tynab** [comment]


**Reply by tauanesales** [reply]
I think I have done all of that, but I haven't received this achievement. This looks like my usual repository creation flow, are there any other steps or something out of the usual that I should pay attention to? Ty!
